server/imports/discord.coffee

```diff
@@ -310,9 +312,11 @@ export class DiscordBot
           channelIdToRoundName[channel.id] = baseName
         # Fetch swarm channel
         # Need string comparison because these numbers are really big
```

Comment:
delete this comment if not true anymore? but maybe change this because it seems accurate still? not sure